relay-server/src/services/outcome.rs

```diff
@@ -205,6 +205,9 @@ impl Outcome {
     /// Returns the `reason` code field of this outcome.
     pub fn to_reason(&self) -> Option<Cow<'_, str>> {
         match self {
+            Outcome::Invalid(DiscardReason::TooLarge(too_large_reason)) => Some(Cow::Owned(
+                format!("too_large_{}", too_large_reason.as_str()),
```

Comment:
What about:

```suggestion
                format!("too_large:{}", too_large_reason.as_str()),
```

The same syntax we use for sampled outcomes.

Reply:
We asked Priscila about this when we made the front-end and sampling is a special case according to here:

> so we are just grouping all of them together and showing 'Dynamic Sampling' you can see it here [https://sentry.io/api/0/organizations/sentry/stats_v2/?category=transaction_index[…]y%29&groupBy=outcome&groupBy=reason&interval=15m&statsPeriod=24h](https://sentry.io/api/0/organizations/sentry/stats_v2/?category=transaction_indexed&field=sum%28quantity%29&groupBy=outcome&groupBy=reason&interval=15m&statsPeriod=24h) and here https://sentry.io/_admin/customers/sentry/?dataType=indexed_transactions&statsPeriod=24h

So hence we did not use the : since that is not shown in the front-end and rather opted for the `_` so that it becomes `Too Large Attachment`, rather than `Too Large: Attachment`.

Reply:
As discussed, we will move forward with `too_large:{reason}`